src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: bold;
+}
+
+.form {
+  margin-top: auto;
```

Comment:
The `margin-top: auto;` on the `.form` class does not seem to serve a purpose unless it is within a flex or grid context with specific alignments.